cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -31,10 +33,71 @@
 import static com.powsybl.cgmes.conversion.Conversion.Config.StateProfile.SSH;
 
 /**
+ * TwoWindingsTransformer Interpretation
+ * <p>
+ * Ratio and Phase Interpretation (Xfmr2RatioPhaseInterpretationAlternative) <br>
+ * END1. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end1 (before transmission impedance) <br>
+ * END2. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end2 (after transmission impedance) <br>
+ * END1_END2. TapChangers (ratioTapChanger and phaseTapChanger) are considered at the end where they are defined in Cgmes <br>
+ * X. If x1 == 0 all tapChangers (ratioTapChanger and phaseTapChanger) are considered at the end1 otherwise they are considered at end2
+ * <p>
+ * Phase Angle Negate. All phaseTapChanger angles are considered -angle.
```

Comment:
I think we have to remove this parameter. In this case, the CGMES file is clearly faulty and our work is to provide a tool coherent with the standard and not a tool to get around it.

Reply:
ok, I will delete this alternative

Reply:
Can you delete the comment too ?